STYLE-GUIDE.md

```diff
@@ -0,0 +1,85 @@
+# Style guide
+
+In order to keep the documentation in this repository consistent and accessible, refer to the style guidelines below
+before contributing documentation.
+
+# Language
+
+## Formal writing voice
+
+All documentation is written in a formal writing voice. For example, avoid the use of contractions or colloquial
+language. Please refer to [this page](http://facultyweb.ivcc.edu/rrambo/tip_formal_writing_voice.htm) for a complete set
+of instructions and examples.
+
+* **Correct**: Do not attempt to solve the problem without consulting the support lead.
+* **Incorrect**: Don't try to fix it without hitting up the support lead.
+
+## General descriptions
+
+General descriptions are written in the passive voice (as opposed to the active voice).
+
+* **Correct**: Design tokens are used for customizing the application theme.
+* **Incorrect**: You can use design tokens to customize your application theme.
+
+## Instructions
+
+Instructions are written in the passive voice or in the second person.
+
+* **Correct**: This module is then imported.
+* **Correct**: You then import this module.
+* **Incorrect**: We then import this module.
+
+# General guidelines
+
+## Release notes
+
+If possible, a release note should link to the documentation page of the related feature. For each release note, try to
+give an example of its use case, and only go into technical detail when necessary.
```

Comment:
Maybe add a line like this: "Any required steps to use the new functionality either go in the migration section if it is a one time action, or should be listed on a page describing the feature so that people can find the usage in later versions."